x-pack/plugins/security_solution/public/timelines/components/timeline/body/renderers/formatted_field.tsx

```diff
@@ -239,6 +241,12 @@ const FormattedFieldValueComponent: React.FC<{
         value={value}
       />
     );
+  } else if (fieldName === RISK_SCORE) {
+    return value ? (
+      <span data-test-subj="risk-score-truncate" title={`${value}`}>
+        {formatRiskScore(Number(value))}
+      </span>
+    ) : null;
```

Comment:
could you add a unit test for this new condition?

Reply:
Heya 100% will get that unit test added, thanks for the feedback.

Reply:
Flyout spot - also yes, thanks for pointing that out.